.github/workflows/run-tests-on-gh.yml

```diff
@@ -93,26 +99,6 @@ jobs:
         if: env.SIMULATION_PRESET == 'standard'
         run: ./mvnw exec:java -Dexec.mainClass=org.performance.parser.TrendDataParser
 
-      - name: Commit Updated Trend Data
-        uses: stefanzweifel/git-auto-commit-action@v5
-        with:
-          commit_message: "chore(perf): Update performance trend data"
-          file_pattern: performance-trends/requests-trend.csv
-          branch: ${{ github.ref_name }}
-
-      - name: Stop the OpenMRS instance
-        if: '!cancelled()'
-        run: docker stop $(docker ps -a -q)
-
-      - name: Upload test results
-        if: always()
-        uses: actions/upload-artifact@v4
-        with:
-          name: report
-          path: target/gatling
-          retention-days: 30
-
```

Comment:
Oder suggestion:

1. Start containers
2. Run tests
3. Stop containers
4. get the previous trend file
5. extract trends
6. Push to the report brach
7. Artifact uploads